src/scheduler/workers/unpublish/weekly/findReminderToSend.ts

```diff
@@ -0,0 +1,27 @@
+import { List } from "@prisma/client";
+import { schedulerLogger } from "scheduler/logger";
+import { ListJsonData } from "server/models/types";
+import { addWeeks, differenceInWeeks, parseISO, startOfDay, startOfToday } from "date-fns";
+
+export function findReminderToSend(list: List) {
+  const logger = schedulerLogger.child({ listId: list.id, method: "findNonRespondentsForList" });
+  const jsonData = list.jsonData as ListJsonData;
+  const { keyDates } = jsonData.currentAnnualReview!;
+  const today = startOfToday();
+  const startDate = startOfDay(parseISO(keyDates.annualReview.START));
+
+  // @ts-ignore
+  const weeksSinceStartDate = differenceInWeeks(today, startDate, { roundingMethod: "floor" });
```

Comment:
1. calculating how many weeks (rounded down) since the start date. 
2. adding this number of weeks to the annual review start date (`reminderToFind`)
  - if the the startDate was a Wednesday, then reminderToFind will also be on a Wednesday
3. use this value to search if a REMINDER has been sent for this week

this is only calculated ONCE per list during runtime, so it is not computationally expensive. 

This is also used to find relevant list items ONCE per list during runtime (i.e. a list item is not checked individually if it has been sent. the query will handle that)